azure_li_services/schema.py

```diff
@@ -0,0 +1,141 @@
+schema = {
+    'version': {
+        'required': True,
+        'type': 'date'
```

Comment:
Is the type `date`? I recall that Robert asked that the date be fixed without dashes to avoid locale interpretation. Or maybe Abhi asked for that, and Robert didn't like it. I honestly don't remember what Robert wanted, but I do remember overruling Abhi and just saying to do it Robert's way! 🙂 

This is really just a placeholder: Be sure that, in the end, the date is implemented as per Robert's preference.

Reply:
Yes, this should just be an 'int' as in 20180614

Reply:
I agree and had it as such in the beginning. Then I realized that the version format is set as in this example:

```
version: 2017-11-15
```

the yaml parser automatically converts this into a date string and if I change the type to int or date it will not validate. So this is the time to explicitly agree on a version format. Let's do that now

Reply:
@rjschwei seems you already had an agreement with Jeff on the version format. I think I missed that. Could you provide this information here ? Thanks